File: WYNK.Data/Model/squinttran.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;
using System.ComponentModel.DataAnnotations.Schema;

namespace WYNK.Data.Model
{
    public class SquintTran
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]

        public int ID { get; set; }
        public int RID { get; set; }
        public Boolean IsDVOD { get; set; }
        public Boolean IsDVOS { get; set; }
        public Boolean IsDVOU { get; set; }
        public DateTime Date { get; set; }
        public int SquintType { get; set; }
        public string SquintDiagnosisDescription { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedUTC { get; set; }
        public int? UpdatedBy { get; set; }
        public DateTime? UpdatedUTC { get; set; }
        public Boolean IsActive { get; set; }


    }
}

File: WYNKWeb/src/app/Models/SquintTran.ts
export class SquintTran {
  ID: number;
  SquintID: number;
  IsDVOD: boolean = false;
  IsDVOS: boolean = false;
  IsDVOU: boolean = false;
  OD: boolean;
  OS: boolean;
  OU: boolean;
  IsActive: boolean;
  chkOD: boolean = true;
  chkOS: boolean = true;
  chkOU: boolean = true;
  checkStatusOD: boolean = false;
  checkStatusOS: boolean = false;
  checkStatusOU: boolean = false;
  SquintType: number = 0;
  SquintDiagnosisDescription = '';
  CreatedBy = 0;
  CreatedUTC: Date;
  UpdatedBy = 0;
  UpdatedUTC: Date;
}


